docs/docs-beta/docs/guides/ingestion-and-transformation/ingesting-data.md

```diff
@@ -0,0 +1,64 @@
+This guide explains how to use Dagster to orchestrate the ingestion of data into a data warehouse or data lake, where it can be queried and transformed. Dagster integrates with several tools that are purpose-built for data ingestion, and it also enables writing custom code for ingesting data.
+
+## What you'll learn
+
+- How Dagster helps with data ingestion
+- How to integrate Dagster with different data ingestion tools
+- How to write custom data ingestion pipelines
+
+<details>
+  <summary>Prerequisites</summary>
+- Familiarity with [asset definitions](/concepts/assets)
+</details>
+
+## How Dagster helps with data ingestion
+
+As a data orchestrator, Dagster helps with data ingestion in the following ways:
+- It can automatically kick off computations that ingest data.
+- It can coordinate data ingestion with downstream data transformation, for example to rebuild a set of dbt models after the upstream data they depend on is updated.
+- It can represent ingested data assets in its data asset graph, which enables understanding what ingested data exists, how ingested data is used, and where data is ingested from.
+
+Note that this guide focuses on batch data ingestion, because streaming data ingestion doesn't typically rely on an orchestrator to kick off or coordinate computations. However, streaming data assets can still be represented in Dagster for lineage purposes.
+
+## Orchestrate a data ingestion tool
+
+Dagster integrates with several tools that are purpose-built for data ingestion. These tools roughly fall into two main categories:
+- Hosted data ingestion services.
+- Embedded data ingestion libraries.
+
+### Hosted data ingestion services
+
+Hosted data ingestion services are tools that provide UIs for configuring and managing syncs between data sources and tables in a data warehouse. Dagster's integrations with these tools help you represent these ingested tables as assets in the Dagster asset graph. And they help you kick off asset materializations that use the REST APIs exposed by these tools to trigger syncs.
+
+Dagster provides two integrations with two hosted data ingestion services:
+- [Fivetran](/guides/ingestion-and-transformation/ingest-data-with-fivetran)
+- [Airbyte](/guides/ingestion-and-transformation/ingest-data-with-airbyte)
+
+### Embedded data ingestion libraries
+
+Embedded data ingestion libraries are tools that enable using code or configuration to manage syncs between data sources and tables in a data warehouse. The source of truth on syncs is typically in a git repository, rather than an external hosted service. Dagster's integrations with these tools help you represent these ingested tables as assets in the Dagster asset graph. And they help you kick off asset materializations that trigger syncs.
+
+Dagster provides two integrations with embedded data ingestion libraries:
+- [Sling](/guides/ingestion-and-transformation/ingest-data-with-sling)
+- [DLT](/guides/ingestion-and-transformation/ingest-data-with-dlt)
```

Comment:
Same suggestion here as I had for the hosted section. I'm not sure you actually need the `Git repositories are...` sentence, as it being in code or configuration sort of implies it. I'm not sure we need to be that specific.

```suggestion
Unlike hosted data ingestion tools, embedded options use code or configuration to manage syncs. Git repositories are typically the source of truth rather than a hosted service.

With Dagster, you can [description of thing you can do with these libraries]

Dagster currently integrates with:

- [Sling](/guides/ingestion-and-transformation/ingest-data-with-sling)
- [DLT](/guides/ingestion-and-transformation/ingest-data-with-dlt)
```